map/navbar.php

```diff
@@ -21,19 +21,19 @@
             } ?>
 
             <li class="nav-item">
-                <a class="nav-link p-2" href="settings.php" rel="noopener" aria-label="Einstellungen" title="Einstellungen"><i class="fas fa-sliders-h"></i></a>
+                <a class="nav-link p-2" href="settings.php" rel="noopener" aria-label="<?php echo L::navbar_settings; ?>" title="<?php echo L::navbar_settings; ?>"><i class="fas fa-sliders-h"></i></a>
             </li>
             <li class="nav-item">
-                <a class="nav-link p-2" href="faq.php" rel="noopener" aria-label="FAQ" title="FAQ"><i class="fas fa-question"></i></a>
+                <a class="nav-link p-2" href="faq.php" rel="noopener" aria-label="<?php echo L::navbar_faq; ?>" title="<?php echo L::navbar_faq; ?>"><i class="fas fa-question"></i></a>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'L'.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4694914)